src/DataSource.ts

```diff
@@ -1,7 +1,8 @@
-import {DataQueryRequest, DataQueryResponse, DataSourceInstanceSettings} from '@grafana/data';
+import {DataQueryRequest, DataQueryResponse, DataSourceInstanceSettings, MetricFindValue} from '@grafana/data';
 import {DataSourceWithBackend, getBackendSrv, getTemplateSrv} from '@grafana/runtime';
-import {FunctionX, Installation, MyDataSourceOptions, MyQuery} from './types';
+import {FunctionX, Installation, MyDataSourceOptions, MyQuery, MyVariableQuery} from './types';
```

Comment:
Perhaps we should try to migrate these types to the ones from node-lynx?

This could be done in a separate PR.